static/index.html

```diff
@@ -56,7 +59,8 @@ <h1>nobody.live</h1>
     <script src="https://embed.twitch.tv/embed/v1.js"></script>
     <script type="text/javascript">
       let numberOfThumbnailsToFetch = 12;
-      let server = 'https://nobody.live';
+      let server =  window.location.hostname === '127.0.0.1' ? 'http://localhost:5000':'https://nobody.live';
+      let trainId = NaN
```

Comment:
Uninitialized variables should usually be left uninitialized/undefined and/or null; setting to NaN (which almost always implies an arithmetic error condition) is not usually the best value to initialize with (even though a variable may technically be NaN).